.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!--
+
+For bug reports, please follow the template below. For enhancement proposals, feel free
+to use whatever template makes sense.
```

Comment:
This seems good, but it doesn't show up currently (when rendered). Any reason for it to be commented?

Reply:
Not really, was just copying another example I'd seen.